client/my-sites/plugins/constants.js

```diff
@@ -7,3 +13,13 @@ export const PREINSTALLED_PLUGINS = [
 	'layout-grid',
 	'page-optimize',
 ];
+
+export const PREINSTALLED_PREMIUM_PLUGINS = {
+	'jetpack-search': {
+		feature: WPCOM_FEATURES_INSTANT_SEARCH,
+		products: {
+			monthly: PRODUCT_JETPACK_SEARCH_MONTHLY,
+			yearly: PRODUCT_JETPACK_SEARCH,
```

Comment:
These products are specific for self-hosted Jetpack sites. Shouldn't we use `PRODUCT_WPCOM_SEARCH` and `PRODUCT_WPCOM_SEARCH_MONTHLY` for WP.com sites?